src/apps/bridge/reportBuilder.cpp

```diff
@@ -323,70 +349,90 @@ void reportBuilderAddToQueue(uint64_t node_id, uint8_t sensor_type, void *sensor
   bool rval = false;
-  switch (sensor_type){
-    case AANDERAA_SENSOR_TYPE: {
-      aanderaa_aggregations_t aanderaa_sample = (static_cast<aanderaa_aggregations_t *>(sensor_data))[sample_index];
-      if (sensor_report_encoder_open_sample(context, AANDERAA_NUM_SAMPLE_MEMBERS) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to open sample in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_add_sample_member(context, encode_double_sample_member, &aanderaa_sample.abs_speed_mean_cm_s) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to add sample member in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_add_sample_member(context, encode_double_sample_member, &aanderaa_sample.abs_speed_std_cm_s) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to add sample member in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_add_sample_member(context, encode_double_sample_member, &aanderaa_sample.direction_circ_mean_rad) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to add sample member in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_add_sample_member(context, encode_double_sample_member, &aanderaa_sample.direction_circ_std_rad) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to add sample member in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_add_sample_member(context, encode_double_sample_member, &aanderaa_sample.temp_mean_deg_c) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to add sample member in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_add_sample_member(context, encode_double_sample_member, &aanderaa_sample.abs_tilt_mean_rad) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to add sample member in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_add_sample_member(context, encode_double_sample_member, &aanderaa_sample.std_tilt_mean_rad) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to add sample member in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_add_sample_member(context, encode_uint_sample_member, &aanderaa_sample.reading_count) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to add sample member in addSamplesToReport\n");
-        break;
-      }
-      if (sensor_report_encoder_close_sample(context) != CborNoError) {
-        BRIDGE_LOG_PRINT("Failed to close sample in addSamplesToReport\n");
-        break;
-      }
-      rval = true;
+  switch (sensor_type) {
+  case AANDERAA_SENSOR_TYPE: {
+    aanderaa_aggregations_t aanderaa_sample =
+        (static_cast<aanderaa_aggregations_t *>(sensor_data))[sample_index];
+    if (sensor_report_encoder_open_sample(context, AANDERAA_NUM_SAMPLE_MEMBERS,
+                                          "aandera_current_v0") != CborNoError) {
```

Comment:
This is the only "real" change in the file. 
The rest is `clang-format`

Reply:
nit: aandera -> aanderaa